AntistasiOfficial.Altis/UI/defines/menu_commander.hpp

```diff
@@ -81,12 +81,43 @@
 
 // INFO MENU
 #define INFO_FIA "['status'] remoteExecCall ['AS_fnc_infoScreen', 2];"
+
+#define INFO_MRESTR "if(server getVariable [""jna_mrestricted"",false]) then {server setvariable [""jna_mrestricted"",flase,true]; jna_minItemMember = [-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1];  (_this select 0) ctrlSetTextColor [0.18,0.545,0.341,1];} \
```

Comment:
I think that he doesn't like the colour thingie here, maybe i'll change it with a simple hint and fuck the green/red